core-infrastructure/src/db/Core.Database/Scripts/065-CreateSchoolExpenditureHistoricWithNullsView.sql

```diff
@@ -0,0 +1,28 @@
+DROP VIEW IF EXISTS SchoolExpenditureHistoricWithNulls;
+
+CREATE VIEW SchoolExpenditureHistoricWithNulls AS
```

Comment:
This is the underlying `VIEW` for most of the below, effectively `NULL`-ing any fields where the corresponding "unit" (i.e. `TotalPupils` or `TotalInternalFloorArea` are themselves `NULL`).

Any operations (i.e. `Avg()` or division) will therefore result in `NULL` for these rows.